website/addons/osfstorage/model.py

```diff
@@ -218,6 +241,20 @@ def path(self):
     def node(self):
         return self.node_settings.owner
 
+    @property
+    def rented(self):
```

Comment:
Because that would return a User object or a None, which are both harder to use for the purposes of renter check. This could/should be renamed to ```rented_by```